pi/interfaces.txt

```diff
@@ -1,4 +1,6 @@
 # refer to https://wiki.debian.org/BridgeNetworkConnectionsProxyArp
+# FLIC_HUB_INTERFACE is the interface connecting the Flic Hub to the RPi (either eth0 or eth1)
```

Comment:
Would you be able to specify what `eth0` vs `eth1` means here?

And what `wlan0` vs `eth0` means in the next line? (I assume that you use `wlan0` for if you want wireless Internet and `eth0` if you want to use ethernet Internet, but it would be nice to have it spelled out.)

Or maybe these options and meanings would be better as comments in the `template.pi_config.ini` since that's actually where someone would need to decide which to use.

Reply:
`eth0` is the builtin ethernet interface on the RPi. `eth1` is the USB-to-ethernet adapter, if present. `wlan0` is the wifi interface.

I've added some brief documentation to `pi/templates/interfaces.txt` explaining this.

Reply:
Thanks!